tests/test_run_steps_filter.py

```diff
@@ -19,6 +19,13 @@ def step(name, cmd=False):
 configs = Configuration()
 """
 
+def get_cli_params(tmpdir):
+    return ["-vt", "none",
+            "-fsd", str(tmpdir),
```

Comment:
Could you please make sure that more conventional ways of getting string representation of this object are still absent?

Reply:
I'm not able to find such ways in the documentation:
https://docs.pytest.org/en/6.2.x/tmpdir.html
https://py.readthedocs.io/en/latest/path.html